README.md

```diff
@@ -370,6 +370,7 @@ Table of Contents
   * [codeship.com](https://codeship.com/) — 100 private builds/month, 5 private projects, unlimited for Open Source
   * [deployhq.com](https://www.deployhq.com/) — 1 project with 10 daily deployments (30 build minutes/month)
   * [drone](https://cloud.drone.io/) - Drone Cloud enables developers to run Continuous Delivery pipelines across multiple architectures - including x86 and Arm (both 32 bit and 64 bit) - all in one place
+  * [GitHub Actions](https://github.com/features/actions) - Build, test, and deploy your code right from GitHub. 
```

Comment:
Unfortunately we prefer to list things once only as this list is already crazy long.

So the existing github entry already lists CI/CD, not perfect but kind of forced by the format to try to keep it under control

thanks for your contribution!